.gitignore

```diff
@@ -112,3 +112,6 @@ venv.bak/
 
 # VSCode Settings
 .vscode
+
+# Webkit's compiling
+frontend/static/frontend/main.js
```

Comment:
⚠️ This needs to be checked in for the deployment to Heroku to work

Reply:
Yeah, we had to change our stance on the `main.js` recently due to deployment. For now, it stays.

Reply:
Ahh cool. I'll remove that from gitignore then.